src/operator/quantization/mkldnn/mkldnn_quantized_elemwise_add.cc

```diff
@@ -135,7 +135,8 @@ static void MKLDNNQuantizedElemwiseAddForward(const nnvm::NodeAttrs& attrs,
     output_data_range = kUint8Range;
     output_data_type  = mkldnn::memory::data_type::u8;
   } else {
-    output_data_range = kInt32Range;
+    // The impact of rounding in line below is negligible.
```

Comment:
Why not just remove this else as the proper values will be already set during initialization?

Reply:
Good point! Thanks